uploadNeuroDB/mass_pic.pl

```diff
@@ -1,5 +1,36 @@
 #!/usr/bin/perl
 
+=pod
+
+=head1 NAME
+
+mass_pic.pl -- Generates check pic for the LORIS database system
```

Comment:
here too; if you decided to rmeove the work 'check' above, you can remove it here as well.
(either way is fine for me)

Reply:
it was there originally but I never understood why it says check pic instead of pic... Removing the check.